CMSIS/DSP/Include/arm_math.h

```diff
@@ -1282,7 +1282,7 @@ __STATIC_INLINE q31_t arm_div_q63_to_q31(q63_t num, q31_t den)
     q31_t   result;
     q63_t   absNum;
     q31_t   normalized;
-    q31_t   norm;
+    int     norm;
```

Comment:
int cannot be used so it should be int32_t
I have a patch coming (with other changes) where this issue is corrected.